doc/source/whatsnew/v0.24.0.txt

```diff
@@ -17,6 +17,10 @@ New features
 
 - ``ExcelWriter`` now accepts ``mode`` as a keyword argument, enabling append to existing workbooks when using the ``openpyxl`` engine (:issue:`3441`)
 
+
+- :meth:`DataFrame.corr` and :meth:`Series.corr` now accept a callable for generic calculation methods of correlation, e.g. histogram intersection
```

Comment:
Use your PR as the issue number.  Also, no new line above this sentence.

Reply:
done